msteams-platform/sbs-graphactivity-feedbroadcast.yml

```diff
@@ -93,193 +93,68 @@ items:
-
-          > [!NOTE]
-          > The **Scope name** should match with the **Application ID** URI with `/access_as_user` appended at the end.</br>
-           `api://bff2****.ngrok.io/00000000-0000-0000-0000-000000000000/access_as_user`.
-
-        1. Set **Who can consent** to **Admins and users**.
-        1. To configure the admin and user consent prompts with appropriate values for **access_as_user** scope, provide the following information in the fields:</br>
-    
-            * Enter **Teams can access the user’s profile** as **Admin consent display name**
+    [!INCLUDE [Azure app registration](includes/get-started/azure-app-registration.md)]
 
-            * Enter **Allows Teams to call the app’s web APIs** as the current user as **Admin consent description**
+    **Create a tunnel**
 
-            * Enter **Teams can access the user profile and make requests on the user’s behalf** as **User consent display name**
+      # [dev tunnel](#tab/dev)
 
-            * Enter **Enable Teams to call this app’s APIs with the same rights as the user** as **User consent description**
-  
-        1. Ensure that **State** is set to **Enabled**.
+      [!INCLUDE [Tunnel](includes/get-started/dev-tunnel.md)]
 
-        1. Select **Add scope** to save.
-
-       Your application ID URI is created with scope name.
-
-       :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/accessasuser.png" alt-text="Screenshot shows the added scope for the application ID URI." border="true":::
-
-    1. In the **Authorized client applications** section, identify the applications that you want to authorize for your app’s web application. 
-    
-    1. Select **Add a client application**. 
-
-       :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/add-a-client.png" alt-text="Screenshot shows the add a client application option." border="true":::
-
-       Add a client application pane appears.
-
-    1. Enter the Client ID for desktop and web application separately that you want to authorize. It's the application ID of client application that you want to authorize.
-       
-       | **Client ID** | **For authorizing** |
-       |--------|-------------|
-       | `1fec8e78-bce4-4aaf-ab1b-5451cc387264` | Teams desktop or mobile application |
-       | `5e3ce6c0-2b1f-4285-8d4b-75ee78787346` | Teams web application |
-
-    1. Select **Authorized scopes** to determine, which type of token return the service.
+      # [ngrok](#tab/ngrok)
+      
+      [!INCLUDE [Tunnel](includes/get-started/ngrok-tunnel.md)]
 
-       :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/add-desktop-app.png" alt-text="Screenshot shows the authorized scope of the client application." border="true":::
+      ---     
 
-       The following image displays the Client ID:
+    [!INCLUDE [Application ID URI](includes/get-started/application-id-uri.md)]
```

Comment:
Update the step as 

In the left pane of Azure portal, .....

Reply:
Not updated as confirmed